src/patient_representation/tl/evaluation.py

```diff
@@ -265,7 +265,7 @@ def evaluate_prediction(y_true, y_pred, task, **parameters):
     return {"score": score, "metric": metric}
 
 
-def test_proportions(target, groups):
+def statistical_test_proportions(target, groups):
```

Comment:
Let's do it like that:

```suggestion
def composition_differences_test(target, groups):
```